### DISCUSSION TITLE
Nextjs failed to compile

### DISCUSSION THREAD

**Discussion by Oluwatob-i** [vercel/next.js#18728] [Help] [answered]
/pages/customer/[id].js
SyntaxError: /home/oluwatobi/Documents/unagarage/pages/customer/[id].js: pragma and pragmaFrag cannot be set when runtime is automatic.
> 1 | /** @jsx jsx */ 
    | ^
  2 | import { jsx, useColorMode } from "theme-ui";
  3 | import { useState, useQuery } from 'react';
  4 | import {useRouter} from 'next/router';

version of Next: 10.0.0
version of Node: 14.13.1
version of react: 17.0.1
os: Linux

**Comment by jpedroschmitz** [comment, marked as answer]
This is not an issue with Next.js, but with Theme UI (see [#1160](https://github.com/system-ui/theme-ui/issues/1160))

To fix this you have two options:

1. You can downgrade React and ReactDOM to 16.13.1 or below;
2. Add a `/** @jsxRuntime classic */` on top of each Theme UI import ([reference](https://github.com/system-ui/theme-ui/issues/1160#issuecomment-715530924)).

Also, you can wait for an update on Theme UI to support the new JSX transform.
